### PR TITLE
Fixed AIs being stunnable

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/-tg-station/issues/14969

### CODE REVIEW

**Comment by neersighted** [comment]
I don't think this is the right method.

`silicon/emp_act()` consists of the following:
```
/mob/living/silicon/emp_act(severity)
	switch(severity)
		if(1)
			src.take_organ_damage(20)
			Stun(8)
		if(2)
			src.take_organ_damage(10)
			Stun(3)
	flick("noise", src:flash)
	src << "<span class='userdanger'>*BZZZT*</span>"
	src << "<span class='danger'>Warning: Electromagnetic pulse detected.</span>"
	..()
```

I don't think its a good idea to allow AIs to be stunned, but thats just me.
If you don't want AIs to be stunned move the stuns to robot.

**Comment by xxalpha** [comment]
Good point, I don't want to nerf AIs.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -1178,4 +1178,11 @@
 		notify_ai(3, oldname, newname)
 	if(camera)
 		camera.c_tag = real_name
-	custom_name = newname
\ No newline at end of file
+	custom_name = newname
+/mob/living/silicon/robot/emp_act(severity)
```
Comment (neersighted) [inline code/modules/mob/living/silicon/robot/robot.dm]: Newline please

Comment (xxalpha) [inline code/modules/mob/living/silicon/robot/robot.dm]: IT WAS THE EDITORS FAULT

**Comment by neersighted** [comment]
:+1: merge it

**Comment by tkdrg** [comment]
conflicts

**Comment by xxalpha** [comment]
No conflicts.

**Pull request merged**